### DISCUSSION TITLE
Is the link to the classicshell mirror on here safe?

### DISCUSSION THREAD

**Discussion by xniwo** [Open-Shell/Open-Shell-Menu#1712] [Q&A]
At the bottom of this github, there are links to the archived classicshell help pages, under coddec github io. Google is blocking the site. Is this just google being over protective regarding the site's potential content, given the original wasn't secure (http)? Or has it found a false positive on the site? 

The mirror is secure (https) but still flagged in the browser (brave), which will let me continue at my own risk with the 'harmful programs not being removed'. The warning itself may put a lot of people off using openshell. Is there any real issue? Thanks

<a href="https://ibb.co/kQF0m1r"><img src="https://i.ibb.co/YpG80ZK/Screenshot-2023-09-23-175812.png" alt="Screenshot-2023-09-23-175812" border="0"></a><br /><a target='_blank' href='https://usefulwebtool.com/convert-lower-upper-case'>upper case converter</a><br />

**Comment by ge0rdi** [comment]
There is nothing wrong with the mirror.
You can check [commits](https://github.com/coddec/Classic-Shell/commits/gh-pages) to the repo. Last one was done back in 2018.
